Gruntfile.js

```diff
@@ -66,7 +66,8 @@ const trans2TSFiles = [
     'src/common/errors.ts',
     'src/common/regex.ts',
     'src/browser/port_discovery.ts',
-    'src/browser/api_protocol/**/**.ts'
+    'src/browser/api_protocol/**/**.ts',
```

Comment:
I just noticed we have this as the first item in the list `'src/browser/api_protocol/api_handlers/clipboard.ts'` which should be removed because of `'src/browser/api_protocol/**/**.ts'`